src/app/components/core/configuration-manager/view-config-item/view-config-item.component.html

```diff
@@ -93,18 +93,15 @@
 
                     <ng-container *ngSwitchCase="'LONG_TEXT'">
                       <ngx-codemirror #jsoneditor name="{{configItem.value.key}}"
-                        [options]="setEditorConfig(configItem.value.type)" [autoFocus]="true"
-                        [ngModel]="JSON.parse(setConfigValue(configItem.value)) | json " (ngModelChange)="checkValidJson(jsoneditor.value)">
+                        [options]="setEditorConfig(configItem.value.type)" [autoFocus]="true" [jsonValue]="isValidJson"
```

Comment:
what is `[jsonValue]` attirbute, looks we are binding it to True and false only?

Reply:
It is custom attribute here, like `required`. And yes, we are binding it to true and false only. This attribute is set to true and false on the basis of valid and invalid input JSON value.
`this.form.controls[key].setErrors({'jsonValue': false});` 
and showing error msg:
```
<span *ngIf="f?.controls[configItem.value.key]?.hasError('jsonValue')" class="help is-danger">
    Invalid JSON
</span>
```

Reply:
then better to rename it as `validJSON`

Reply:
This is the name of directive, and changing its name effects on many components. So it is better to keep as it is.